app/views/content/is-teaching-right-for-me/physics/_article.html.erb

```diff
@@ -0,0 +1,120 @@
+<div class="row inset">
+<section class="col col-720">
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
```

Comment:
@caitlin-reidy think we need to add a bit more in about why it's an exciting career choice. But we can leave it for now and add it to a ticket in the backlog.